documentation/design/proposals/bridging_external.adoc

```diff
@@ -27,6 +29,8 @@ spec:
   - name: remote1
     hostname: example.com
     port: 5672
+    failoverUrls:
```

Comment:
As we now have the ability to express failover url(s), doesn't that make host and port redundant?

Reply:
Yeah, I felt this was perhaps a bit weird. The reason i made it a url was to follow the router syntax which also does this. However, maybe we should change this to:

```
failoverHosts:
- hostname: failover.example.com
  port: 5672
```

Then the scheme will be restricted to the same as the original host (i.e. same tls settings).

Come to think of it, maybe change `hostname` to `host` both for failoverHosts and the connector `hostname` field?

Reply:
The point I was trying to make was that if we've got a list, then having separate hostname/port is redundant.  We could just enforce that the list has at least one entry.  Maybe `endpointHosts` would be a better name than `failoverHosts`.

I think host/port is better than hostname/port as there is then consistency with the external auth service.

Reply:
Ah, that is a good point. And in terms of router configuration, we will use the first entry for the main connection.